packages/immutable-rootfs/30cos-immutable-rootfs/module-setup.sh

```diff
@@ -29,7 +29,7 @@ install() {
     # Include utilities required for cos-setup services,
     # probably a devoted cos-setup dracut module makes sense
     inst_multiple -o \
-        partprobe lsblk sgdisk mkfs.ext2 mkfs.ext3 mkfs.ext4 mkfs.vfat mkfs.fat mkfs.xfs blkid e2fsck resize2fs mount xfs_growfs umount
+        blockdev lsblk sgdisk mkfs.ext2 mkfs.ext3 mkfs.ext4 mkfs.vfat mkfs.fat mkfs.xfs blkid e2fsck resize2fs mount xfs_growfs umount
```

Comment:
I just thought at it. But this is the kind of changes that forces the package to depend on a version of yip >= 0.9.22 (in runtime)

Reply:
good point! wait for tests to pass then addd it + force-merge? 

Not only the installer but also the inmutablefs package should have a minimum yip dependency as part of this